src/index.ts

```diff
@@ -44,8 +45,10 @@ const canPass = {
   // eslint-disable-next-line
   signTx(
     tx: { actions: Array<any> },
-    callback?: (error: any, data?: any) => any
+    callback?: (error: any, data?: any) => any,
+    ots?: RequestSignTxOptions,
```

Comment:
@duyphanz I think it is better to:

* change name from `ots` to `signTxOption`
* move  `signTxOption` to be before `callback`